source/vscode/src/changelog.ts

```diff
@@ -42,27 +44,37 @@ export async function maybeShowChangelogPrompt(
       "qdk.lastChangelogNotificationVersion",
       CHANGELOG_VERSION,
     );
-    // We don't want to show the prompt if this is a freash install of the QDK as that is a bit too noisy.
-    if (lastChangelogNotificationVersion !== undefined) {
-      const buttons = ["What's New?", "Don't show this again"];
-      const choice = await vscode.window.showInformationMessage(
-        "The Azure Quantum Development Kit has been updated.",
-        ...buttons,
-      );
-      if (choice === buttons[0]) {
-        await vscode.commands.executeCommand("qsharp-vscode.showChangelog");
-      } else if (choice === buttons[1]) {
-        await vscode.workspace
-          .getConfiguration("Q#")
-          .update(
-            "notifications.suppressUpdateNotifications",
-            true,
-            vscode.ConfigurationTarget.Global,
-          );
-        vscode.window.showInformationMessage(
-          'You will no longer receive "What\'s New" notifications. You can re-enable them from the Q# settings.',
+    const buttons = ["What's New?", "Don't show this again"];
+    const associatedId = getRandomGuid();
+    sendTelemetryEvent(EventType.ChangelogPromptStart, {
+      associationId: associatedId,
+      qdkVersion: CHANGELOG_VERSION,
```

Comment:
Not to be nitpicky, but this should be "changelogVersion" as the property name. As discussed, we'll only bump the CHANGELOG_VERSION when we have notable new changes, so there may be months where we ship a new QDK but don't bump this if there's only minor fixes and updates.

Also, the VS Code telemtry sends through the extension version anyway, so we always get that (i.e. all events have fields like `"ExtensionName":"quantum.qsharp-lang-vscode","ExtensionVersion":"1.18.0"`)

Reply:
done